lib/tizen_sdk.dart

```diff
@@ -276,17 +299,9 @@ class TizenSdk {
             'The arm64 build is not supported by the $profile profile.');
         profile = 'iot-headed';
```

Comment:
This conversion is invalid (overrides the "tizen" rootstrap) and actually is no longer necessary.

Reply:
I changed the code for this condition: Check `arch` and change only `type`.